nowplaying/input/handler.py

```diff
@@ -1,12 +1,7 @@
-#! /usr/bin/env python
```

Comment:
I'm removing these shebang/__version__/$Id$ shenanigans all over the place. They are either obsolete with python 3 or didn't work from the get go.